extract.js

```diff
@@ -0,0 +1,59 @@
+const i18n = require('i18n-extract');
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash').mixin(require('lodash-keyarrange'))
+const i18nFolder = path.join(__dirname, 'src/i18n');
+
+const keys = i18n.extractFromFiles([
+  'src/**/*.js',
+], {
+  marker: 'polyglot.t',
+});
+
+function _mergeMissing(transls, missing, makeVal) {
+  _.each(missing, (i) => {
+    if (transls[i.key] === undefined) {
+      transls[i.key] = makeVal(i);
+    }
+  });
+}
+
+function _writeTransls(file, transls) {
+  const sorted = _.keyArrange(transls)
+  fs.writeFileSync(file, JSON.stringify(sorted, {}, 2));
```

Comment:
I'm not talking about asynchronous/await syntax sugar, I'm talking about using `fs.writeFile` instead of `fs.writeFileSync`.